frame/randomness-collective-flip/src/lib.rs

```diff
@@ -220,7 +220,7 @@ mod tests {
 		type OnSetCode = ();
 	}
 
-    impl pallet_randomness_collective_flip::Config for Test {} 
+    impl pallet_randomness_collective_flip::Config for Test {}
```

Comment:
Still a space problem here

Reply:
It seems that there is a problem with my vim config, which has been fixed